### PR TITLE
Book Details component added

### PR DESCRIPTION
**Pull request by shubhwale**

### CODE REVIEW

File: src/app/books/book-details/book-details.component.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  constructor(public bookService: BookService,config: NgbRatingConfig,private route: ActivatedRoute) {
+    config.readonly = true;
+    config.max=5;
+  }
+
+  book: Book
+  
+  ngOnInit(): void {
+    this.route.paramMap.subscribe((map: ParamMap) => {
+      let bookId = +map.get("id");
```
Comment (ketan-wellsky) [inline src/app/books/book-details/book-details.component.ts:24]: what does this +map.get("id") do?

Comment (ketan-wellsky) [reply src/app/books/book-details/book-details.component.ts:24]: Also, you have a merge conflict. Try merging the master in this branch and do the PR again

Comment (ketannaik) [inline src/app/books/book-details/book-details.component.ts:24]: What does +map.get("id") do?

Comment (ketannaik) [reply src/app/books/book-details/book-details.component.ts:24]: Also, you branch has merge conflict. Please merge the master in this branch and push the changes that should resolve it.

File: src/app/books/book-details/book-details.component.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                </div>
+                <div class="col-md-6">
+                    <table class="table table-bordered">
+                        <tr>
+                            <td>Book Id</td>
+                            <td>{{ book.bookId }}</td>
+                        </tr>
+                        <tr>
+                            <td>Title</td>
+                            <td>{{ book. title }}</td>
```
Comment (bala0105) [inline src/app/books/book-details/book-details.component.html:24]: Any specific reason for space  <td>{{ book. title }}</td> ?

File: src/app/books/book-details/book-details.component.ts (hunk 1 above)
Comment (shubhwale) [reply src/app/books/book-details/book-details.component.ts:24]: We import ParamMap and Activated route class for paramMap function.
map.get("id") gets id parameter from uri route which is defined in app-routing.module.ts. Adding + converts it into integer

Comment (shubhwale) [reply src/app/books/book-details/book-details.component.ts:24]: merge conflict solved

File: src/app/books/book-details/book-details.component.html (hunk 2 above)
Comment (shubhwale) [reply src/app/books/book-details/book-details.component.html]: That was a mistake. Solved

**Pull request merged**